Tasks/ServiceFabricDeploy/task.json

```diff
@@ -87,7 +87,7 @@
             "name": "useDiffPackage",
             "type": "boolean",
```

Comment:
wont there be task.loc.json changes as well?

Reply:
udpated